### PR TITLE
feat(NRQL): changed the file structure and new page titles

### PR DESCRIPTION
**Pull request by jeff-colucci**
Revamping the left nav for NRQL. Tasks:

- Change location of files in left nav
- Rename all NRQL help files for more clarity and consistency
- Add redirects
- Change existing links in docs to new links
- Create new TOC

### CODE REVIEW

**Comment by austin-schaefer** [comment]
Moving PR review to @akristen so @jeff-colucci has feedback in time next week

File: src/content/docs/nrql/nrql-examples/browserspa-nrql-query-examples.mdx [hunk 1]
```diff
@@ -17,6 +17,7 @@ redirects:
   - /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/new-relic-browserspa-nrql-query-examples
   - /docs/query-spa-data-insights
   - /docs/browser/single-page-app-monitoring/use-spa-data/query-spa-data-new-relic
+  - /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/browserspa-nrql-query-examples
```
Comment (akristen) [inline src/content/docs/nrql/nrql-examples/browserspa-nrql-query-examples.mdx:20]: I don't think this is in the yml file. when I click it in the nav, it disappears

File: src/nav/nrql.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      - title: Use subqueries for linking multiple queries
+        path: /docs/nrql/using-nrql/subqueries-in-nrql
+      - title: Use subquery joins to combine queries
+        path: /docs/nrql/using-nrql/subquery-joins
+  - title: NRQL examples
+    pages:
+      - title: App data NRQL examples
+        path: /docs/nrql/nrql-examples/app-data-nrql-query-examples
+      - title: Browser/SPA NRQL examples
+        path: /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/browserspa-nrql-query-examples
```
Comment (akristen) [inline src/nav/nrql.yml:69]:
```suggestion
        path: /docs/nrql/nrql-examples/browserspa-nrql-query-examples/
```

File: src/nav/nrql.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      - title: New Relic query builder
+        path: /docs/query-data/nrql-new-relic-query-language/query-tools/chart-builder-create-query-visualizations
+      - title: NerdGraph (GraphQL API)
+        path: /docs/query-data/nrql-new-relic-query-language/query-tools/nrql-api-queries
+  - title:  Explore NRQL
+    pages:
+      - title: Set a query's time range
+        path: /docs/nrql/using-nrql/query-time-range
+      - title: Query metric data
+        path: /docs/telemetry-data-platform/get-data/apis/query-metric-data-type
```
[the remark this answers is not in the document]
Comment (akristen) [reply src/nav/nrql.yml:41]: It is a bit jarring to be sent out of the NRQL category -- it definitely confused me when it first happened. Not sure if there's a solution tho

Comment (akristen) [inline src/nav/nrql.yml:41]:
```suggestion
        path: /docs/telemetry-data-platform/get-data/apis/query-metric-data-type
```

re: the comments, this link

**Review by akristen — CHANGES_REQUESTED** [review]
There is one thing in the YML file that needs to be updated. Once that gets fixed, I'll approve! There's one other doc that i had a Q about, but if that link is intentional then don't worry about it.

File: src/nav/nrql.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jeff-colucci) [reply src/nav/nrql.yml:41]: Yeah, these are pre-existing dummy docs and we decided to keep them as they were for now, it's not ideal but it mirrors existing behavior

[the remark this answers is not in the document]
Comment (akristen) [reply src/nav/nrql.yml:41]: OK! That makes sense. I think it's fine :)

**Review by akristen — APPROVED** [review]
That link works! I'm approving. LMK if you ever revisit this work. I think it could be fun as a squad to think through the "Explore NRQL" category. IMO this is great and already a big improvement -- i could not tell you where our nrql docs were prior to this

**Pull request merged**